<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EchangeArmList</class>
 <widget class="QWidget" name="EchangeArmList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="lw_blueArmList">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>170</y>
     <width>320</width>
     <height>320</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="lb_blueName">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>100</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>人物名字</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lb_blueIcon">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>50</y>
     <width>321</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>人物头像</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_blueListBackground">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>170</y>
     <width>320</width>
     <height>320</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../Resource/Photo/actionlist.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>lb_blueListBackground</zorder>
  <zorder>lw_blueArmList</zorder>
  <zorder>lb_blueIcon</zorder>
  <zorder>lb_blueName</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
